client-js/app/index.html

```diff
@@ -19,7 +19,7 @@
   <!-- Add to homescreen for Chrome on Android -->
   <meta name="mobile-web-app-capable" content="yes">
   <meta name="application-name" content="LabRAD">
-  <link rel="icon" sizes="192x192" href="/images/touch/chrome-touch-icon-192x192.png">
+  <link rel="icon" href="/images/labrad.png">
```

Comment:
While I like the general idea, I find that being just black and white with the outlines of the L and the R makes this look indistinct and almost like a Chinese character (particularly when viewed at a vertical angle and against the gray background of a tab or Bookmarks bar). I would instead propose separating the L and R and coloring them differently. Perhaps something like that below (feel free to use this if you wish):
![labrad2](https://cloud.githubusercontent.com/assets/8183919/13543836/16128d5c-e224-11e5-9007-8a67f4fc3bda.png)